### PR TITLE
Attributes for menuBarTemplate

### PR DESCRIPTION
**Pull request by MarhyCZ**
Hi, it's a very small commit which adds option to set attributes onto the menuBarTemplate tag. Thanks to this, I was able to set theme: 'dark' attribute.
darkmenu
lightmenu

### CODE REVIEW

File: src/menu.js [hunk 1]
```diff
@@ -10,12 +10,14 @@ let created = false;
...
 let doc = Parser.dom(docStr);
 let menuBarEl = (doc.getElementsByTagName('menuBar')).item(0);
+let menuBarTpl = (doc.getElementsByTagName('menuBarTemplate')).item(0);
 let menuBarFeature = menuBarEl && menuBarEl.getFeature('MenuBarDocument');
 let itemsCache = {};
 
 // default menu options
 let defaults = {
     attributes: {},
+    attributesTpl: {},
```
Comment (emadalam) [inline src/menu.js]: The name `attributesTpl` is a bit confusing as it somehow gives an impression that we are passing a template that would be converted to attributes 🙅‍♂️  How about using `rootTemplateAttributes` to indicate the intent more precisely 🥅

File: src/menu.js [hunk 2]
```diff
@@ -97,7 +99,8 @@ function addItem(item = {}) {
  *
  * @example
  * ATV.Menu.create({
- *     attributes: {},
+ *     attributes: {},  // menuBar attributes
+ *     attributesTpl: {}, // menuBarTemplate attributes
```
Comment (emadalam) [inline src/menu.js]: Can you also update the readme files sample code in the https://github.com/emadalam/atvjs#creating-menu-page section to indicate the same update 💪

**Comment by MarhyCZ** [comment]
Hi @emadalam, of course, I will happily do the changes. I wasn't convinced about the name either 💯 .

**Comment by MarhyCZ** [comment]
Ok, I have done the proposed changes :)

**Pull request merged**